### PR TITLE
Fix #139: Wrong scope for dependencies

### LINKED ISSUE #139: Wrong  scope for dependencies in maven pom file  io.arrow-kt:arrow-fx-kotlinx-coroutines:0.10.5
The newly released io.arrow-kt:arrow-fx-kotlinx-coroutines:0.10.5 doesn't seem to have a correct pom file.
It declares dependencies to three libraries:
- io.arrow-kt:arrow-fx:0.10.5
- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.61
- org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.3
Unfortunately those 3 dependencies are declared in scope "runtime" which should have been "compile". So, this means that a project that depends on arrow-fx-kotlinx-coroutines needs to re-declare those dependencies in the correct scope.

### PR DESCRIPTION
**Pull request by rachelcarmena**
Fix #139 issue about the wrong scope for arrow-fx-kotlinx-coroutines dependencies.

### CODE REVIEW

File: arrow-fx-kotlinx-coroutines/build.gradle [hunk 1]
```diff
@@ -14,9 +14,9 @@ apply from: "$DOC_CONF"
 apply from: "$PUBLISH_CONF"
 
 dependencies {
-    implementation project(':arrow-fx')
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
+    compile project(':arrow-fx')
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
+    compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
```
Comment (aballano) [inline arrow-fx-kotlinx-coroutines/build.gradle:19]:
```suggestion
    api "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
```

File: arrow-fx-kotlinx-coroutines/build.gradle [hunk 2]
```diff
@@ -14,9 +14,9 @@ apply from: "$DOC_CONF"
 apply from: "$PUBLISH_CONF"
 
 dependencies {
-    implementation project(':arrow-fx')
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
+    compile project(':arrow-fx')
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
```
Comment (aballano) [inline arrow-fx-kotlinx-coroutines/build.gradle:18]:
```suggestion
    api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
```

File: arrow-fx-kotlinx-coroutines/build.gradle [hunk 3]
```diff
@@ -14,9 +14,9 @@ apply from: "$DOC_CONF"
 apply from: "$PUBLISH_CONF"
 
 dependencies {
-    implementation project(':arrow-fx')
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$KOTLIN_VERSION"
-    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$KOTLINX_COROUTINES_VERSION"
+    compile project(':arrow-fx')
```
Comment (aballano) [inline arrow-fx-kotlinx-coroutines/build.gradle:17]:
```suggestion
    api project(':arrow-fx')
```

**Comment by rachelcarmena** [comment]
Thanks for the review, @aballano :raised_hands: Right, `api` replaces the old `compile` in this case :+1:

**Pull request merged**